.travis.yml

```diff
@@ -5,9 +5,23 @@ php:
   - 5.5
   - 5.6
   - 7.0
+  - 7.1
```

Comment:
I'd say yes. Since this is a core component, i think we should run the tests against as much versions as possible.

But i agree that this change should have probably been introduced in another PR.